src/app/core/components/movie-statistics/movie-statistics.component.html

```diff
@@ -5,9 +5,17 @@
   </div>
   <span class="stat-container main">
     <span class="total-amount">
-      {{ statistics.amountInCollection }}
+      {{ statistics.amountInCollection | number: '1.0-0' : 'en-US' }}
       {{ statistics.amountInCollection === 1 ? 'Movie' : 'Movies' }}
     </span>
+    <span class="total-time">
+      {{ runtimeInHours | number: '1.0-0' : 'en-US' }}
+      {{ runtimeInHours === 1 ? 'hour watched' : 'hours watched' }}
```

Comment:
time to think about a generalization because you're repeating code quite often is a very predictive way.
Maybe not generalize everything but a few components shared across the different features?